interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,451 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="interfaces">
+    <children>
+      <tagNode name="tinc" owner="${vyos_conf_scripts_dir}/interfaces-tinc.py">
+        <properties>
+          <help>Tinc VPN Tunnel Interface</help>
+          <priority>460</priority>
+          <constraint>
+            <regex>^tinc[0-9]+$</regex>
+          </constraint>
+          <constraintErrorMessage>Tinc VPN tunnel interface must be named vtincN</constraintErrorMessage>
+          <valueHelp>
+            <format>tincN</format>
+            <description>Tinc VPN interface name</description>
+          </valueHelp>
+        </properties>
+        <children>
+          <leafNode name="node-name">
+            <properties>
+              <help>Local Node Name options(require)</help>
```

Comment:
Why capitalize "Node Name"? Also, it should be `[REQUIRED]` as per out convention, although I'm not a big fan of it — if users miss it, the commit verification will fail and they will know, so I'd remove it altogether.

Reply:
What's the meaning?